tool/tsh/tsh.go

```diff
@@ -231,6 +231,7 @@ func Run(args []string) {
 	app.Flag("enable-escape-sequences", "Enable support for SSH escape sequences. Type '~?' during an SSH session to list supported sequences. Default is enabled.").
 		Default("true").
 		BoolVar(&cf.EnableEscapeSequences)
+	app.Flag("bind-addr", "Address in the form of host:port to bind to for login command webhook").Envar(bindAddrEnvVar).StringVar(&cf.BindAddr)
```

Comment:
I personally find the naming of this confusing, since we are mixing implementation details with the flag.

```
bind-addr", "Override default host:port for tsh login"
```
^ Open to feedback?

Reply:
Yeah, it's confusing. The thing is that we're going to be triggering the `tsh login` flow automatically if someone is logged out and they run `tsh ls` or `tsh clusters` for example, so it's hard to explain it. There's also the fact that users who don't use SSO will never even see a loopback listener.

Reply:
Maybe 

```
`bind-addr", "Override default host:port for explicit port and host mapping"
```

Reply:
I think that's even more vague...!

Reply:
I think that's ok, I'm mainly trying to drop the 'webhook' terminology.

It seems the two main use cases for this are 
- Connecting to docker container on mac https://github.com/gravitational/teleport/issues/2620
- Changing the default port / host for SSO logins

Reply:
How about `Override host:port used when opening a browser for cluster logins`?

More explicit about exactly what it does and removes "webhook".